style.css

```diff
@@ -0,0 +1,527 @@
+  height: 352px;
+  width: 369px;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+  flex: 1;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.txt1 {
+  margin-top: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.lang-title {
+  margin-bottom: 120px;
+}
+
+.tags1 {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  column-gap: 5px;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  margin-top: -90px;
+  gap: 8px;
+  width: 271px;
+  height: 40px;
+  order: 1;
+}
+
+.tag1 {
+  padding: 5px;
+  border: 1px solid rgb(224, 224, 224);
+  margin-bottom: 5px;
+  border-radius: 8px;
+  font-size: 12px;
+  line-height: 19px;
+  font-weight: 600px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  color: rgb(6, 202, 137);
+}
+
+.tags2 {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  margin-top: 60px;
+}
+
+.grid-image {
+  height: 30px;
+  width: 30px;
+  background-color: white;
+  border-color: 1px gray;
+  padding: 10px;
+  border-radius: 100px;
+  margin-top: -3px;
+}
+
+p.content {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  margin-left: 150px;
+  margin-right: auto;
+  width: 45em;
+  padding: 15px;
+  margin-top: 100px;
+}
+
+.about-text > h2 {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  position: relative;
+  padding-left: 2rem;
+  padding-right: 2rem;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 20px;
+  gap: 24px;
+  position: absolute;
+  width: 1171px;
+  left: 80px;
+  height: auto;
+}
+
+.form-text {
+  align-items: center;
+  text-align: center;
+}
+
+.form-text > h2 {
+  font-family: "Inter", sans-serif;
+  font-size: 30px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 24px;
+  height: 330px;
+  flex: none;
+  order: 1;
+}
+
+ul {
+  list-style: none;
+}
+
+form li + li {
+  margin-top: 1em;
+}
+
+input {
+  font: 1em "inter", sans-serif;
+  width: 650px;
+  height: 48px;
+  box-sizing: border-box;
+}
+
+textarea {
+  font: 1em "inter", sans-serif;
+  width: 650px;
+  box-sizing: border-box;
+  border: 1px solid green;
+  vertical-align: top;
+  height: 114px;
+}
+
+input:focus:valid,
+textarea:focus:valid {
+  border-color: blue;
+}
+
+button {
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: white;
+  border: none;
+  padding: 12px;
+  background: #36b37f;
+  border-radius: 4px;
+  order: 3;
+  margin-top: 20px;
+}
+
```

Comment:
- [ ] Please follow the rules about [media queries ](https://developer.mozilla.org/es/docs/Web/CSS/Media_Queries/Using_media_queries) to make your design a desktop version. I'm attaching it next to this line as an example. But it's should be applied to the whole page. 💯 💯 💯

***Remember that by using media queries you can manipulate better your different sizes on the page*** ❗ 

Your design:

![image](https://user-images.githubusercontent.com/93630700/199714070-029ac488-1ace-4773-a346-e851cb7e80c7.png)

FIGMA design:

![image](https://user-images.githubusercontent.com/93630700/199714390-fd84d469-29d1-496c-a399-13768cae25d8.png)